packages/sui-js/src/string/to-query-string.js

```diff
@@ -9,11 +9,12 @@ import {stringify} from 'qs'
  * @param {string} [options.delimiter] - delimiter
  */
 function toQueryString(queryParams, options = {}) {
-  const {arrayFormat, delimiter} = options
+  const {arrayFormat, delimiter, encode} = options
```

Comment:
I believe that in this case we must put a default value because we are creating our API. So we do not leave that default responsibility to the library on we use, in this case `qs`.

```suggestion
  const {arrayFormat, delimiter, encode = true} = options
```

Reply:
+1